Yoast/wordpress-seo
# Fix metabox when opening on text view after refactoring

## Summary

The order was messed up because of the refactoring.

## Relevant technical choices:

* Initializing tinyMCE can safely be done after initializing the plugins.

## Test instructions

This PR can be tested by following these steps:

* Open the post/page edit page
* Go to the text tab
* Save the post/page
* The analysis should work
